bazel/repositories.bzl

```diff
@@ -94,7 +94,7 @@ py_library(name = "envoy_repo", srcs = ["__init__.py"], visibility = ["//visibil
 _envoy_repo = repository_rule(
     implementation = _envoy_repo_impl,
     attrs = {
-        "envoy_root": attr.label(default = "@envoy//:BUILD"),
+        "envoy_version": attr.label(default = "@envoy//:VERSION.txt"),
```

Comment:
this change was necessary, otherwise changing the version doesnt expire the cache and the version is set incorrectly